server/src/testFixtures/java/io/crate/testing/LogicalPlanAssert.java

```diff
@@ -47,6 +47,13 @@ public LogicalPlanAssert isEqualTo(String expectedPlan) {
         return this;
     }
 
+    public LogicalPlanAssert hasOperators(String ... operator) {
```

Comment:
Opposed to the `isEqualTo` variant, this generates error messages where the missing/unexpected elements are highlighted which makes it eaiser to spot the difference.